api/filesystem/listfiles.md

```diff
@@ -0,0 +1,11 @@
+# listfiles
+**Signature:** `listfiles(folderPath: string): {string}` <br>
```

Comment:
The return type should be: {[number]: string}

Reply:
{string} is actually valid shorthand syntax in Luau for {[number]: string}, you can check the docs if you like.

Reply:
See https://luau-lang.org/typecheck#table-indexers

Reply:
I can't believe I wrote all that code without knowing this!